plugins/core/webview/src/q-ui/components/login.vue

```diff
@@ -81,6 +81,7 @@ export default defineComponent({
             this.mutateStage('START')
         },
         login(type: LoginOption) {
+            window.ideApi.postMessage({command: 'sendTelemetry'})
```

Comment:
since message to java is sent from `@login` so you don't need to add @sendTelemetry actually

Reply:
so we could revert `loginOptions.vue`, `qOptions.vue`, `toolkitOptions.vue`

Reply:
yeah, I had a different command for sendTelemetry earlier